incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -33,18 +35,33 @@ var (
 
 var treeCmd = &cobra.Command{
 	Use:   "tree",
-	Short: "Displays the hierarchy tree rooted at the given namespace",
-	Args:  cobra.ExactArgs(1),
+	Short: "Display one or more hierarchy trees",
 	Run: func(cmd *cobra.Command, args []string) {
-		nnm := args[0]
+		flags := cmd.Flags()
 		footnotesByMsg = map[string]int{}
 		footnotes = []string{}
-		hier := getHierarchy(nnm)
-		fmt.Println(nameAndFootnotes(hier))
-		printSubtree("", hier)
-
+		var nsList []string
```

Comment:
You don't need this line; on line 45 you could just say:

`nsList := args`

which initializes the new variable to the same type as `args`.